types/src/v0/impls/auction.rs

```diff
@@ -29,11 +29,24 @@ impl FullNetworkTx {
     }
 }
 
-impl Committable for BidTxBody {
+impl Committable for BidTx {
     fn tag() -> String {
         "BID_TX".to_string()
     }
 
+    fn commit(&self) -> Commitment<Self> {
+        let comm = committable::RawCommitmentBuilder::new(&Self::tag())
+            .field("body", self.body.commit())
+            .fixed_size_field("signature", &self.signature.into());
+        comm.finalize()
+    }
+}
+
+impl Committable for BidTxBody {
+    fn tag() -> String {
+        "BID_TX_BODY".to_string()
+    }
+
```

Comment:
It doesn't appear in the diff, but on line `57` there is another call to `bincode`.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2194/commits/23f640c710e737319a6cff0580af793319ec38b7